system/views/core/navigation/mainNavigation.cfm

```diff
@@ -5,18 +5,18 @@
 	liCurrentClass          = args.liCurrentClass          ?: 'active';
 	liHasChildrenClass      = args.liHasChildrenClass      ?: 'dropdown';
 	liHasChildrenAttributes = args.liHasChildrenAttributes ?: '';
-	aCurrentClass           = args.aCurrentClass           ?: 'active';
-	aHasChildrenClass       = args.aHasChildrenClass       ?: '';
-	aHasChildrenAttributes  = args.aHasChildrenAttributes  ?: '';
+	aCurrentClass           = args.aCurrentClass           ?: '';
+	aHasChildrenClass       = args.aHasChildrenClass       ?: 'dropdown-toggle';
+	aHasChildrenAttributes  = args.aHasChildrenAttributes  ?: 'data-toggle="dropdown"';
 </cfscript>
 
 <cfoutput>
 	<cfloop array="#menuItems#" index="i" item="item">
-		<cfset hasChildren = item.children.len() />
-		<li class="<cfif item.active>#liCurrentClass#</cfif><cfif hasChildren> #liHasChildrenClass#</cfif>" <cfif hasChildren>#liHasChildrenAttributes#</cfif>>
-			<a class="<cfif item.active>#aCurrentClass#</cfif><cfif hasChildren> #aHasChildrenClass#</cfif>" href="#event.buildLink( page=item.id )#" <cfif hasChildren>#aHasChildrenAttributes#</cfif>>#item.title#</a>
+		<cfset hasChildren = not item.children.isEmpty() />
```

Comment:
This change makes me curious, what does `not item.children.isEmpty()` give us that `item.children.len()` does not? (other than a "true" boolean rather than an integer)?